benchalerts/clients.py

```diff
@@ -209,9 +273,13 @@ def get_comparison_to_baseline(self, contender_sha: str) -> dict:
         comparisons = {}
         log.info(f"Getting comparisons from {len(all_runs)} runs")
         for run in all_runs:
-            comparison = self.get(
-                f"/compare/runs/{run['baseline']['run_id']}...{run['contender']['run_id']}"
+            path = (
+                "/compare/runs/"
+                f"{run['baseline']['run_id']}...{run['contender']['run_id']}"
             )
+            if z_score_threshold:
+                path += f"?threshold_z={z_score_threshold}"
```

Comment:
I'd rather `.get()` pass a `params` argument through to requests

Reply:
I always forget about that!

Reply:
💯 great catch